Crashlytics/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [changed] Removed obsolete API calls from upload-symbols
+- [changed] Decreased Crashlytics minimum deployment target from iOS 10 to iOS 9
```

Comment:
"Decreased Crashlytics CocoaPods minimum deployment target from iOS 10 to iOS 9"

since other distribution mechanisms will stay at iOS 10.